### PR TITLE
edit docstrings

### PR DESCRIPTION
**Pull request by aurbeez**
created to add the missing event docstrings but can be used to double check and add anything else to the docstrings if needed. @Cedric-Boucher

### CODE REVIEW

File: src/app/src/main/java/com/example/myapplication/Event.java [hunk 1]
```diff
@@ -19,7 +19,13 @@ public class Event {
     private QRCode qrCode;
     private final EntrantPool entrantPool;
 
-
+    /***
+     * Private base constructor to consolidate code used by other constructors
```
Comment (Cedric-Boucher) [inline src/app/src/main/java/com/example/myapplication/Event.java:23]: says private but the code says public

**Review by Cedric-Boucher — CHANGES_REQUESTED** [review]

**Review by Cedric-Boucher — APPROVED** [review]

**Comment by Cedric-Boucher** [comment]
thanks @aurbeez!

**Comment by ichandel** [comment]
Looks Good

**Review by ichandel — APPROVED** [review]
looks good!

**Pull request merged**